src/Categories/Functor/CartesianClosed.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Functor.CartesianClosed where
+
+open import Level
+
+open import Categories.Category.CartesianClosed.Bundle using (CartesianClosedCategory)
+open import Categories.Category.CartesianClosed
+open import Categories.Functor using (Functor; _∘F_)
+open import Categories.Functor.Cartesian
+
+import Categories.Morphism as M
+
+private
+  variable
+    o ℓ e o′ ℓ′ e′ : Level
+
+record IsCartesianClosedF (C : CartesianClosedCategory o ℓ e) (D : CartesianClosedCategory o′ ℓ′ e′)
```

Comment:
`C` and `D` can probably be implicits.

Reply:
It turns out that, since the type of `F` involves `U C` instead of just `C`, Agda can't infer `C` and `D`.

Reply:
Ok!